front/app/modules/commercial/customizable_navbar/admin/components/NavbarItemForm.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Similar to the `PageForm` component, but only has a field for the navbar item title. Used for 'default' navbar items, like 'All input' or 'Events'